src/srvctl/main.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2022, KNS Group LLC (YADRO).
+ */
+#include "config.h"
+
+#include "srvctl.hpp"
```

Comment:
Why `srvctl` and not clio?
It's a CLI Organizer (and a Greek muse of history, daughter of Zeus and Mnemosine, the goddess of memory), it won't just 'ctl' the 'srv', it will also 'ctl' the 'bmc', and do everything else eventually.

Reply:
`srvctl` will a subcommand of global `clio`. I'm going to do this later when I add/backport other stuff into this project.